profiles.js

```diff
@@ -1,16 +1,122 @@
 "use strict";
-// orchil/profiles.js
 var profiles = {
```

Comment:
I'm assuming that you're adding all these so we can unify the different games into a single profiles.js? That's reasonable - not better or worse including all the HTML clients the way we currently do!